test/run_tests.sh

```diff
@@ -41,8 +37,7 @@ if docker build -f test/Dockerfile -t kudo-test .; then
     if docker run -e INTEGRATION_OUTPUT_JUNIT --net=host -it --rm -m 4g \
         -v /var/run/docker.sock:/var/run/docker.sock \
         -v "$(pwd)"/reports:/go/src/github.com/kudobuilder/kudo/reports \
-        -v "$(pwd)"/kind-logs:/tmp/kudo-e2e-test \
-        kudo-test bash -c "make $TARGET; ret=\$?; chmod a+r -R /tmp/kudo-e2e-test; exit \$ret"
+        kudo-test bash -c "make $TARGET; ret=\$?; chmod a+r -R /go/src/github.com/kudobuilder/kudo/reports; exit \$ret"
```

Comment:
👏 
Good to know! The reason for the `/tmp/kudo-e2e-test` was the requirement for absolute paths and not knowing where the reports would be written by default.